### ISSUE TITLE
WordPress upgrade failed. Site files have system attribute set on Windows

### ISSUE THREAD

**Issue by Steveorevo** [ServerPress/ds-cli#6]
#### STEPS TO REPRODUCE:
1. On a Windows system create a new website with DS-CLI enabled.
2. Launch cmd.exe and navigate to the website's document root.
3. type attrib, files (readme.html) will appear with the system attribute set.
#### RESULT:

Upgrading WordPress core can fail, often with an error on writing or copying, i.e. readme.html
#### EXPECTED RESULT:

Don't set system attribute bit on non-symbolic linked files.

Browser:  N/A
Browser Version:  N/A
Operating System: Windows 7, 8, 10

**Comment by Steveorevo** [comment]
This issue occurs within prepend_1.php's lines 59,60,61:

```
        if ( $ds_runtime->last_ui_event->action === 'site_created' || $ds_runtime->last_ui_event->action === 'site_imported' ) {
            ds_cli_exec( "symfix " . $ds_cli_site_path );
        }
```

We can remove the lines as it's rare to have symbolic links inside one's project. Or we can fix the symfix script at: 

https://github.com/ServerPress/ds-cli/blob/master/platform/win32/pre/symfix

Anyone good with *nix/regular expressions? This shouldn't have touched the readme.html file much less any files whereby the first 512 characters do not contain the phrase `!<symlink>`

???

**Comment by Steveorevo** [comment]
Addressed in commit ac481d2774f6b3e3a6a872b9ae4c50429bb43f15

**Comment by GreggFranklin** [comment]
I believe the issue has been resolved.
